Data/participants.json

```diff
@@ -22,4 +22,12 @@
       "githubLink": "https://github.com/yk1ax"
     }
   ]
+  {
+      "name": "Akshat Bhardwaj ",
+      "email": "akshatb622@gmail.com",
```

Comment:
This is not correct. The data inside `{}` should be inside the `[]`.Please see other merged PRs for example.
You can read more about the `JSON` syntax here: https://www.digitalocean.com/community/tutorials/an-introduction-to-json